imports/api/knowledgebase/server/methods.js

```diff
@@ -0,0 +1,178 @@
+import { check } from 'meteor/check';
+import { ValidatedMethod } from 'meteor/mdg:validated-method';
+import { ErxesMixin } from '/imports/api/utils';
+import { KbTopics, KbCategories, KbArticles } from '../collections';
+import { KbTopicsSchema, KbCategoriesSchema, KbArticlesSchema } from '../schema';
+
+// add
+export const addKbTopic = new ValidatedMethod({
+  name: 'knowledgeBaseTopics.add',
+  mixins: [ErxesMixin],
+
+  validate({ doc }) {
+    check({ ...doc, createdBy: this.userId, createdDate: new Date() }, KbTopicsSchema);
```

Comment:
Create extra schema for auto fields like createdBy, createdDate. See /imports/api/forms/form.js and /imports/api/forms/server/methods.js files. Please follow the common pattern.